tests/Context/ServerOsContextTest.php

```diff
@@ -169,13 +169,13 @@ public function offsetSetDataProvider(): array
                 'kernel_version',
                 1,
                 InvalidOptionsException::class,
-                'The option "kernel_version" with value 1 is expected to be of type "string", but is of type "integer".',
+                '/The option "kernel_version" with value 1 is expected to be of type "string", but is of type "(integer|int)"./',
```

Comment:
As this is a regex now, you should escape all special characters like the dot. Also, please use `^` and `$` to make sure that the matching is done against the string as a whole